src/thirdparty/dukglue/dukvalue.h

```diff
@@ -336,7 +336,7 @@ class DukValue {
 	inline duk_int_t as_int() const {
 		if (mType != NUMBER)
 			throw DukException() << "Expected number, got " << type_name();
-		return static_cast<uint32_t>(mPOD.number);
+		return static_cast<int32_t>(mPOD.number);
```

Comment:
Ideally, we should upstream this fix if it hasn’t been already.

Reply:
What do you mean by that?

Reply:
It's a bug in a third party library, we should send a PR to it.
https://github.com/Aloshi/dukglue/blob/master/include/dukglue/dukvalue.h

Reply:
Ah OK. Should I revert my changes here then? It has been fixed upstream already.
https://github.com/Aloshi/dukglue/blob/c3ec2a463a03a4fa34715aa75273537d39900edb/include/dukglue/dukvalue.h#L336~L340

Reply:
No, you should keep it. The library is not updated automatically. Or even manually. I went back to the commit when IntelOrca first added DUK and the bug was already present there.